.github/workflows/e2e.yml

```
@@ -78,6 +78,20 @@ jobs:
         env:
           NODE_ENV: "production"
 
+      - name: Build C3 package for npm
+        run: cd packages/create-cloudflare && npm pack
```

I think you can use [pack-destination](https://docs.npmjs.com/cli/v7/commands/npm-pack#pack-destination) to avoid the copying step below.

```suggestion
        run: cd packages/create-cloudflare && npm pack --pack-destination $HOME
```